asf_heat_pump_suitability/utils/save_utils.py

```diff
@@ -28,3 +29,29 @@ def save_to_s3(df: pl.DataFrame, path: str) -> None:
             "Save to S3 can only save .parquet or .csv file types."
             "Please ensure the `path` argument contains one of these file types."
         )
+
+
+def upload_file_to_s3(
+    local_file_path: str,
+    s3_bucket: str,
+    s3_key_dir: str,
+    save_to_s3: bool,
+    filename: str,
+    subfolder: str,
+):
+    """
+    Upload a local file to an S3 bucket, used for evaluation of our heat network suitability model using the DESNZ Heat Network pilot zones.
```

Comment:
```suggestion
    Upload a local file to an S3 bucket.
```
Just to make the docstring more generic to match the function :)